main.go

```diff
@@ -59,13 +62,34 @@ func main() {
 			go newService.Boot()
 		}
 
+		var storage microstorage.Storage
+		{
+			storage, err = memory.New(memory.DefaultConfig())
+			if err != nil {
+				panic(err)
+			}
+		}
+
+		var transactionResponder transaction.Responder
```

Comment:
Here a transaction responder is configured which is needed by our microstorage framework.